### ISSUE TITLE
MIDI Notes out of range

### ISSUE THREAD

**Issue by OGPanzerhund** [richie0866/MidiPlayer#7]
I've noticed that on some MIDI files, it plays notes that are outside of the 75 key Roblox piano. Is there a chance that a fix could come out, where it transposes out-of-range octaves to the Roblox 75 key input? It would make things sound a lot better without those janky missing notes, and probably a lot more believable too.

MIRP, another MIDI player which is under a creative commons attribution license (meaning it can be copied from, as long as credit is due, I believe. I'm not too familiar with American laws), happens to do this pretty well, but I'm unsure if it's programmed in Lua or something else.

**Comment by danyalassi** [comment]
Yeah coz it isnt possible to do that and u gotta find midi that has bit of keys
